Makefile

```
@@ -1,4 +1,4 @@
-TARGET_IMAGES = avr cortex-m risc-v
+TARGET_IMAGES = cortex-m-openocd
```

Nope, just add your new build target to the list.

```suggestion
TARGET_IMAGES = avr cortex-m risc-v cortex-m-openocd
```

---

wups, i removed those for testing and diddnt add trhem back :dagger: 
my bad